compute/instances/how-to/migrate-routed-ips.mdx

```diff
@@ -13,11 +13,11 @@ categories:
   - compute
 ---
 
-During deployment, each Instance gets a NAT IP from its hosting node, besides its public IP, which is routed to the Instance through a [carrier-grade NAT](/compute/instances/concepts/#carrier-grade-nat-(cgnat)) solution. However, if the underlying hypervisor changes, the NAT IP changes. This can happen when you restart your Instance. The NAT IP lets Instances communicate via an underlying internal network, without public IPs.
+During deployment, each Instance created before April 2024 used to get a NAT IP from its hosting node, besides its public IP, which is routed to the Instance through a [carrier-grade NAT](/compute/instances/concepts/#carrier-grade-nat-(cgnat)) solution. However, if the underlying hypervisor changes, the NAT IP changes. This can happen when you restart your Instance. The NAT IP lets Instances communicate via an underlying internal network, without public IPs.
 
-Routing a public IP directly to an Instance bypasses NAT IPs, sending traffic to its [routed flexible IP](/compute/instances/concepts/#routed-flexible-ip). The benefit is the IP stays consistent even if the Instance is restarted or moved. Routed IPs also let you assign multiple flexible IPv4s and IPv6s to an Instance, as both IPv4 and IPv6 are supported.
+Routing a public IP directly to an Instance bypasses NAT IPs, sending traffic to its [routed flexible IP](/compute/instances/concepts/#routed-flexible-ip). The benefit is the IP stays consistent even if the Instance is restarted or moved. Routed IPs also let you assign multiple flexible IPv4s and IPv6s to an Instance, as both IPv4 and IPv6 are supported (see below for details).
```

Comment:
```suggestion
Since October 2024 it is no no longer possible to create Instances with NAT IPs. Instead, Instances' public IPs are **routed** IPs. Routing a public IP directly to an Instance bypasses NAT IPs, sending traffic to its [routed flexible IP](/compute/instances/concepts/#routed-flexible-ip). The benefit is the IP stays consistent even if the Instance is restarted or moved. Routed IPs also let you assign multiple flexible IPv4s and IPv6s to an Instance, as both IPv4 and IPv6 are supported.
```